app/models/log_count.rb

```diff
@@ -0,0 +1,5 @@
+class LogCount < ActiveRecord::Base
+    scope: two_months_ago do
+        where(created_at: 2.months.ago)
+    end
+end
```

Comment:
I did not think about that, I totally just made extra work for myself, I wanted to use an existing model, but I didn't want to mess anything up while doing it, I'll refactor as suggested.